dist/bin/kctf-cluster

```diff
@@ -398,8 +398,8 @@ function start_operator_gce {
   SUFFIX=$(echo "${PROJECT}-${CLUSTER_NAME}-${ZONE}" | sha1sum)
   POLICY_NAME="kctf-policy-${SUFFIX:0:16}"
   # restart the operator with the new range
-  "${KCTF_BIN}/yq" eval "(.spec.template.spec.containers[0].env[] | select(.name == \"ALLOWED_IPS\")).value = \"${RANGES}\"" "${KCTF_CTF_DIR}/kctf/resources/operator.yaml" \
-    | "${KCTF_BIN}/yq" eval "(.spec.template.spec.containers[0].env[] | select(.name == \"SECURITY_POLICY\")).value = \"${POLICY_NAME}\"" - \
+  "${KCTF_BIN}/yq" eval "(((.|select(has(\"spec\"))).spec|select(has(\"template\"))).template.spec.containers[0].env[] | select(.name == \"ALLOWED_IPS\")).value = \"${RANGES}\"" "${KCTF_CTF_DIR}/kctf/resources/operator.yaml" \
```

Comment:
what does this do? :)

Reply:
`operator.yaml` now has multiple yaml objects on it (before it only had one). so now we have to select the right one =(

Reply:
ah, ok. thanks :)